src/gc/sample/gcenv.ee.cpp

```diff
@@ -330,3 +324,8 @@ MethodTable* GCToEEInterface::GetFreeObjectMethodTable()
 {
     return g_pFreeObjectMethodTable;
 }
+
+bool GCToEEInterface::CreateThread(void (*threadStart)(void*), void* arg, bool is_special, const wchar_t* name)
+{
+    return nullptr;
```

Comment:
This method returns bool...

Reply:
ack, you're right - this is left over from a previous refactor.